src/components/autocomplete/Autocomplete.vue

```diff
@@ -27,6 +27,10 @@
                 v-show="isActive && (data.length > 0 || hasEmptySlot)"
```

Comment:
I think you should show header slot whent there is no data too.
For example add a new computed prop `hasHeaderSlot` and add it to `|| hasEmptySlot`.